charts/castai-agent/templates/rbac.yaml

```diff
@@ -279,4 +289,46 @@ subjects:
     namespace: {{ .Release.Namespace }}
 {{- end}}
 
+{{- if (gt (len .Values.rbac.configmapsReadAccessNamespaces) 0) }}
```

Comment:
nitpick: is length check needed? I would expect `range` behavior would not change

```
{{range pipeline}} T1 {{end}}
	The value of the pipeline must be an array, slice, map, or channel.
	If the value of the pipeline has length zero, nothing is output;
	otherwise, dot is set to the successive elements of the array,
	slice, or map and T1 is executed. If the value is a map and the
	keys are of basic type with a defined order, the elements will be
	visited in sorted key order.
```

Reply:
it should not, it's just a defensive reflex I guess. removing it